.github/workflows/release.yml

```
@@ -0,0 +1,39 @@
+on:
+  release:
+    types:
+      - created
+
+jobs:
+  aws-release:
+    name: Release zip file for Lambda deployment
```

So it doesn't get truncated in the Actions view:
```suggestion
    name: Release zip for AWS Lambda
```